system-test/testnet-performance/testnet-automation.sh

```diff
@@ -133,6 +133,15 @@ function launchTestnet() {
   echo --- configure database
   net/init-metrics.sh -e
 
+  echo --- fetch reusable testnet keypairs
+  if [[ ! -d net/keypairs ]] ; then
+    git clone git@github.com:solana-labs/testnet-keypairs.git net/keypairs
+    # If we have provider-specific keys (CoLo*, GCE*, etc) use them instead of generic val*
+    if [[ -d net/keypairs/${CLOUD_PROVIDER} ]] ; then
```

Comment:
@mvines , `remote-node.sh` doesn't know which cloud provider it's running on, but it might be nice for the multi-cloud nets to have pubkeys like `gce1, msft1, aws2` etc. rather than all internal nodes as `val1, val2`  wdyt?

Reply:
sure, that'd be fun